ocaml/idl/datamodel_cluster.ml

```diff
@@ -28,8 +28,8 @@ let create = call
     ~doc:"Creates a Cluster object and one Cluster_host object as its first member"
     ~result:(Ref _cluster, "the new Cluster")
     ~versioned_params:
-            ([{param_type=Ref _network; param_name="network"; param_doc="the single network on which corosync carries out its inter-host communications"; param_release=kolkata_release; param_default=None};
-             {param_type=String; param_name="cluster_stack"; param_doc="simply the string 'corosync'. No other cluster stacks are currently supported"; param_release=kolkata_release; param_default=None};
+            ([{param_type=(Ref _pif); param_name="PIF"; param_doc="The PIF to connect the cluster's first cluster_host to"; param_release=kolkata_release; param_default=None};
```

Comment:
I think this needs `lima_release`.

Reply:
Should that be available in the datamodel? It currently isn't.

Reply:
You will need something like this: https://github.com/edwintorok/xen-api/commit/7ee65030c021da19eea1741174047df6bc424b93
You can cherry-pick that commit if you like